torch/csrc/jit/codegen/cuda/lower_utils.cpp

```diff
@@ -36,13 +36,42 @@ kir::IfThenElse* cloneIfThenElse(kir::IfThenElse* ite) {
 
 namespace ir_utils {
 
-TVDomainGuard::TVDomainGuard(TensorView* _tv, TensorDomain* td)
-    : tv_(_tv), prev_domain(tv_->domain()) {
+TVDomainGuard::TVDomainGuard(TensorView* tv, TensorDomain* td)
+    : tv_(tv), prev_domain_(tv_->domain()) {
   tv_->setDomain(td);
 }
 
+TVDomainGuard::TVDomainGuard(TVDomainGuard&& guard)
```

Comment:
nit: should we delete copy constructor?

Reply:
deleted in https://github.com/csarofeen/pytorch/pull/1953/files#diff-5760d90c9f11c9ac3eaeb65d7ab72f4008ada59e0688f05cf77bdd6dc4d28c0dR49